tests/useIsMobile.test.js

```diff
@@ -1,11 +1,14 @@
 import { ResponsiveProvider, useIsMobile } from '../src';
+import { breakpoints, breakpointsMax } from './__fixtures__/mockBreakpoints';
```

Comment:
In this file you forgot to change the describe to `useIsMobile`

Reply:
done!